<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>599</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AC升级工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>网元信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>954</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAddNe">
           <property name="text">
            <string>添加网元</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonDelNe">
           <property name="text">
            <string>删除网元</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonAddVersionFile">
           <property name="text">
            <string>版本文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVersionFile">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>版本信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVersion">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTableView" name="tableViewNe"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxCheckOption">
      <property name="title">
       <string>升级选项</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxActiveSoft">
         <property name="text">
          <string>7.激活版本文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBoxUpdateVersionFile">
         <property name="text">
          <string>5.上传版本文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxSaveNeConfigToLocal">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>4.保存配置到本地</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxReboot">
         <property name="text">
          <string>8.复位</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxCheckUpdateFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>2.检测版本文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBoxSaveNeConfig">
         <property name="text">
          <string>3.保存当前运行配置</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBoxUpgradeSoft">
         <property name="text">
          <string>6.释放版本文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBoxAfterRebootConnectTest">
         <property name="text">
          <string>9.复位后连通性检测</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxCheckNe">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1.检测网元</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxAfterRebootTest">
         <property name="text">
          <string>10.升级后检测</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkBoxfterRebootConfigCheck">
         <property name="text">
          <string>11.升级后配置检测</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>升级操作</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonCheckNe">
         <property name="text">
          <string>网元检测</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QTextEdit" name="textEditInformation">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonUpdateAll">
         <property name="text">
          <string>升级全部</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>取消操作</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionImportConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportConfig">
   <property name="text">
    <string>导入配置</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>保留配置</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存配置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
